lib/Bio/EnsEMBL/DataCheck/Checks/UniProtDisplayXref.pm

```diff
@@ -0,0 +1,59 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::UniProtDisplayXref;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'UniProtDisplayXref',
+  DESCRIPTION    => 'Genes display_xref are attached to Uniprot/*, instead of UniProtKB Gene Name (Uniprot_gn)',
```

Comment:
```suggestion
  DESCRIPTION    => 'Gene display xrefs are only attached to UniProtKB Gene Names (Uniprot_gn)',
```
The description needs to be inverted from what I presume was in the healthcheck, to state what the result of a pass means.